content/automate/ManageIQ/Transformation/Common.class/__methods__/assesstransformation.rb

```diff
@@ -69,6 +69,9 @@ def main
             }
             @handle.set_state_var(:factory_config, factory_config)
 
+            # Store source VM power state, as we well power it off
```

Comment:
@fdupont-redhat I think you probably meant to say, "....as we will power it off"